data/persons.txt

```diff
@@ -611,3 +611,161 @@ ship "Modified Osprey"
 	explode "small explosion" 40
 	explode "tiny explosion" 28
 	"final explode" "final explosion large" 1
+
+
+person "Local God"
+	government "Author"
+	frequency 200
+	personality
+		mining harvests surveillance unconstrained forebearing
+	phrase
+		phrase
+			"friendly author"
+	phrase
+		word
+			"Search for Ursa Polaris Station!"
+	phrase
+		word
+			"Someday"
+			"One day"
+			"Eventually"
+		word
+			", "
+		word
+			"maybe I should "
+			"I might "
+			"I will "
+			"I'm going to "
+		word
+			"park this heap"
+			"moor this beast"
+			"put down roots"
+			"find a cozy orbit"
+		word
+			" and "
+		word
+			"retire as an outfit dealer."
+			"open a univeral outfitter."
+			"start dealing exotic outfits."
+			"sell outfits from everywhere."
+			"stock outfits from all over the place."
+	phrase
+		word
+			"If only "
+			"I wish "
+			"Reputable Captains wish "
+		word
+			"someone "
+			"a rich Captain "
+			"an entrepreneurial spirited individual "
+			"a Captain with wealthy friends "
+		word
+			"would "
+		word
+			"build "
+			"construct "
+			"put together "
+			"cobble together "
+			"make "
+		word
+			"and stock "
+		word
+			"a universal outfitter space station."
+			"a place you could buy any outfit."
+			"somewhere you could optimize your ships, without cris-crossing the known galaxy."
+			"a space station out of asteroids - that carried all the outfits."
+	phrase
+		word
+			"I "
+		word
+			"stole "
+			"captured "
+			"restored "
+			"found "
+			"got a hold of "
+			"appropriated "
+			"procured "
+ 		word
+			"this very old Korath World Ship and "
+		word
+			"made it my own"
+			"customized it"
+			"made some modifications"
+			"made a few modifications"
+		word
+			", with a little help, of course."
+	ship "Ursa Polaris" "Local God"
+		"never disabled"
+
+ship "Ursa Polaris"
+	sprite "ship/localworldship"
+	attributes
+		category "Heavy Warship"
+		cost 142857
+		shields 39000
+		hull 27000
+		"required crew" 559
+		"bunks" 972
+		"mass" 1839
+		"drag" 12
+		"heat dissipation" .9
+		
+		"fuel capacity" 600
+		"solar collection" 15
+		"afterburner thrust" 200
```

Comment:
If you have built in afterburner thrust, then I believe you should also be able to have a built in afterburner sound and effect.

Reply:
i probably could, but i actually wanted the extra thrust to be subtle

Reply:
The afterburner thrust of a normal afterburner is only 25, but here you have 200. Since there's no afterburner heat, energy, or fuel, the afterburner will always be in use when in combat. And with only a drag of 12, I believe that means that this ship has a max speed of 1217 while out of combat and 2217 while in combat.

That seems a bit more than subtle to me. :)

Reply:
all intentional - heat damage is very difficult to balance against, and pretty much all other ships fare poorly against it - this ship was designed to be resistant to it, and yet is not indestructible (or "heat-proof" - since there's no such thing with flamethrowers).

in practice, because of the mining personality, afterburners will nearly always be in use - i usually play with the sound off, but Power of the People's spawn is always obvious because of the ionic afterburner plume (which is nearly always present when they are)

if you think the ship is too fast, i'm open to slowing it down, but the ship is supposed to attract attention (and perhaps point people at a long standing and popular plugin which may or may not ever get pulled for the main branch - which is part of the point), and further PotP's "Boss Ship" motif for [Author] government ships.

Reply:
It's fine by me, I'm just saying that it isn't something I'd consider subtle.

Reply:
fair enough - to me, having a ship operate with a max speed of 1217 in very rare instances rather than 2217 most of the time seemed pretty subtle in my play testing - almost unnoticeable, actually.

Reply:
>i'll also note that the multiple thrusters have more to do with the appearance of the flare(s)

Yeah, I guess as much when flying this around in game. Looks great!